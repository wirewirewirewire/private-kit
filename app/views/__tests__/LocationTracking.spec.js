import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import LocationTracking from '../LocationTracking';

it('renders correctly', () => {
  const tree = renderer
    .create(<LocationTracking />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
