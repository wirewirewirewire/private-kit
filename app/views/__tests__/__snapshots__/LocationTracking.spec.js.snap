// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#FFF",
      "color": "#000",
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "center",
    }
  }
>
  <RCTScrollView
    contentContainerStyle={
      Object {
        "alignItems": "center",
        "flex": 1,
        "flexDirection": "column",
        "justifyContent": "space-between",
        "width": "80%",
      }
    }
  >
    <View>
      <Menu
        style={
          Object {
            "alignSelf": "flex-end",
            "position": "absolute",
            "zIndex": 10,
          }
        }
      >
        <MenuTrigger
          style={
            Object {
              "marginTop": 14,
            }
          }
        >
          <Image
            source={
              Object {
                "testUri": "../../../app/assets/images/kebabIcon.png",
              }
            }
            style={
              Object {
                "height": 28,
                "padding": 14,
                "width": 15,
              }
            }
          />
        </MenuTrigger>
        <MenuOptions>
          <MenuOption
            onSelect={[Function]}
          >
            <Text
              style={
                Object {
                  "fontFamily": "OpenSans-Regular",
                  "fontSize": 14,
                  "padding": 10,
                }
              }
            >
              Licenses
            </Text>
          </MenuOption>
        </MenuOptions>
      </Menu>
      <Text
        style={
          Object {
            "fontFamily": "OpenSans-Bold",
            "fontSize": 38,
            "padding": 0,
            "textAlign": "center",
          }
        }
      >
        Private Kit
      </Text>
      <View
        style={
          Object {
            "flex": 6,
            "justifyContent": "space-around",
          }
        }
      >
        <Image
          source={
            Object {
              "testUri": "../../../app/assets/images/PKLogo.png",
            }
          }
          style={
            Object {
              "alignSelf": "center",
              "height": 164.4,
              "marginTop": 12,
              "opacity": 0.3,
              "width": 132,
            }
          }
        />
        <View
          accessible={true}
          focusable={true}
          isTVSelectable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignSelf": "center",
              "backgroundColor": "#665eff",
              "borderRadius": 12,
              "height": 52,
              "justifyContent": "center",
              "opacity": 1,
              "width": 589.9499999999999,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#ffffff",
                "fontFamily": "OpenSans-Bold",
                "fontSize": 14,
                "letterSpacing": 0,
                "lineHeight": 19,
                "textAlign": "center",
              }
            }
          >
            START LOGGING
          </Text>
        </View>
        <Text
          style={
            Object {
              "fontFamily": "OpenSans-Regular",
              "fontSize": 12,
              "lineHeight": 24,
              "marginLeft": 10,
              "marginRight": 10,
            }
          }
        >
          NOTE: After clicking this button you may be prompted to grant Private Kit access to your location.
        </Text>
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "flex": 2,
            "flexDirection": "row",
            "justifyContent": "space-between",
            "marginBottom": -10,
            "width": 589.9499999999999,
          }
        }
      >
        <View
          accessible={true}
          focusable={true}
          isTVSelectable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#454f63",
              "borderRadius": 8,
              "height": 76,
              "justifyContent": "center",
              "opacity": 1,
              "width": 172.5,
            }
          }
        >
          <Image
            resizeMode="contain"
            source={
              Object {
                "testUri": "../../../app/assets/images/export.png",
              }
            }
            style={
              Object {
                "height": 21.6,
                "width": 32.2,
              }
            }
          />
          <Text
            style={
              Object {
                "color": "#ffffff",
                "fontFamily": "OpenSans-Bold",
                "fontSize": 12,
                "letterSpacing": 0,
                "lineHeight": 17,
                "marginTop": 6,
                "opacity": 0.56,
                "textAlign": "center",
              }
            }
          >
            Import
          </Text>
        </View>
        <View
          accessible={true}
          focusable={true}
          isTVSelectable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#454f63",
              "borderRadius": 8,
              "height": 76,
              "justifyContent": "center",
              "opacity": 1,
              "width": 172.5,
            }
          }
        >
          <Image
            resizeMode="contain"
            source={
              Object {
                "testUri": "../../../app/assets/images/export.png",
              }
            }
            style={
              Array [
                Object {
                  "height": 21.6,
                  "width": 32.2,
                },
                Object {
                  "transform": Array [
                    Object {
                      "rotate": "180deg",
                    },
                  ],
                },
              ]
            }
          />
          <Text
            style={
              Object {
                "color": "#ffffff",
                "fontFamily": "OpenSans-Bold",
                "fontSize": 12,
                "letterSpacing": 0,
                "lineHeight": 17,
                "marginTop": 6,
                "opacity": 0.56,
                "textAlign": "center",
              }
            }
          >
            Export
          </Text>
        </View>
        <View
          accessible={true}
          focusable={true}
          isTVSelectable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#454f63",
              "borderRadius": 8,
              "height": 76,
              "justifyContent": "center",
              "opacity": 1,
              "width": 172.5,
            }
          }
        >
          <Image
            resizeMode="contain"
            source={
              Object {
                "testUri": "../../../app/assets/images/newspaper.png",
              }
            }
            style={
              Object {
                "height": 21.6,
                "width": 32.2,
              }
            }
          />
          <Text
            style={
              Object {
                "color": "#ffffff",
                "fontFamily": "OpenSans-Bold",
                "fontSize": 12,
                "letterSpacing": 0,
                "lineHeight": 17,
                "marginTop": 6,
                "opacity": 0.56,
                "textAlign": "center",
              }
            }
          >
            News
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "fontSize": 12,
            "fontWeight": "600",
            "justifyContent": "flex-end",
            "padding": 4,
            "paddingBottom": 10,
            "textAlign": "center",
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "fontFamily": "OpenSans-Regular",
                "fontSize": 12,
                "lineHeight": 24,
                "marginLeft": 10,
                "marginRight": 10,
              },
              Object {
                "paddingTop": 15,
                "textAlign": "center",
              },
            ]
          }
        >
          For more information visit the Private Kit homepage:
           
        </Text>
        <Text
          onPress={[Function]}
          style={
            Array [
              Object {
                "fontFamily": "OpenSans-Regular",
                "fontSize": 12,
                "lineHeight": 24,
                "marginLeft": 10,
                "marginRight": 10,
              },
              Object {
                "color": "blue",
                "marginTop": 0,
                "textAlign": "center",
              },
            ]
          }
        >
          privatekit.mit.edu
        </Text>
      </View>
    </View>
  </RCTScrollView>
</RCTSafeAreaView>
`;
